src/olympia/addons/templates/addons/impala/button.html

```diff
@@ -48,7 +50,16 @@
       {{ _('End-User License Agreement') }}
     </a>
   {% endif %}
-
+  {% if waffle.switch('webext-permissions') %}
+    <br>
+    <a class="webext-permissions badge" href="#">
+      {% if version and not version.all_files[0].is_webextension %}
```

Comment:
Maybe I'm being paranoid but I'd add a check for `version.all_files` being truthy. Never know in what weird context the template might be used...